cinder/scheduler/filters/sap_affinity_filter.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2014, eBay Inc.
```

Comment:
Ah, probably that should be "2022, SAP SE"

Reply:
fixed